changelog/_unreleased/2024-09-20-move-navigationpageloadedevent-to-end.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Move NavigationPageLoadedEvent to end
+issue: NEXT-000000
+author: Niklas Wolf
+author_email: wolfniklas94@web.de
+author_github: @niklaswolf
+---
+# Core
+* move NavigationPageLoadedEvent so that the canonical can be changed via event-subscriber
```

Comment:
The important thing here is, that a changelog entry has to begin with a word like `Changed` ([see here](https://github.com/shopware/shopware/blob/trunk/adr/2020-08-03-implement-new-changelog.md#changelog-entries)), i.e.
```suggestion
* Changed `NavigationPageLoader` to dispatch the `NavigationPageLoadedEvent` after setting the canonical URL, such that the canonical can be changed via an event-subscriber
```

Reply:
Thanks 🙏